extension/scripts/content.js

```diff
@@ -18,18 +18,21 @@ chrome.runtime.onMessage.addListener((request, sender, sendResponse) => {
 
 		const observer = new MutationObserver((mutations) => {
 			mutations.forEach((m) => {
-				// console.log('Mutation: ', m.type)
+				// console.log('Mutation className: ', m.addedNodes[0].className)
 				if (
 					m.type === 'childList' &&
 					m.addedNodes.length > 0 &&
+					m.addedNodes[0] &&
 					m.addedNodes[0].className === '' &&
```

Comment:
This line was just refining the statement and avoiding errors like can't read ".class" of undefined